### DISCUSSION TITLE
How to plot dashed lines using pyvista

### DISCUSSION THREAD

**Discussion by lxmedai** [pyvista/pyvista#5041] [Q&A] [answered]
I wounder if pyvista supports rendering lines as dashed lines in general? I've found "pyvista.Chart2D.border_style" which supports it, but I cannot find any other examples where this is applicable.

**Comment by banesullivan** [comment, marked as answer]
The line cells themselves (the actually line geometry) would need to "dashed". AFAIK, VTK does not support "line styles"

**Reply by lxmedai** [reply]
Thanks a lot for the clarification!

**Comment by olly-writes-code** [comment]
This would be so nice. But I can't see VTK ever supporting this (as per this [discussion](https://discourse.vtk.org/t/vtk-js-dashed-lines/10745)). So it would work on the Pyvista side to create something akin to dashed lines (via splitting lines) or a deeper upstream change in VTK which seems unlikely.

**Comment by user27182** [comment]
> The line cells themselves (the actually line geometry) would need to "dashed". AFAIK, VTK does not support "line styles"

A new `dashed_lines` filter is added in https://github.com/pyvista/pyvista/pull/9292 to process the line cells. And a separate `Actor.line_style` property is also added, which dashes line in rendering only (using shader replacements). Should be available next version, 0.50
